src/redash_stmo/datasource_link/web/datasource-link.html

```diff
@@ -0,0 +1,2 @@
+<a ng-if="dataSource.options.doc_url != '' && dataSource.options.doc_url" ng-href={{dataSource.options.doc_url}}>{{dataSource.type_name}} documentation</a>
+<span ng-if="dataSource.options.doc_url == '' || !dataSource.options.doc_url">{{ dataSource.type_name }} documentation</span>
```

Comment:
This needs a tiny bit of update given the difference in what the datasource link API endpoint returns.